default/stringtables/en.txt

```diff
@@ -3891,6 +3891,9 @@ Word Title Matches
 ENC_SEARCH_PARTIAL_MATCHES
 Partial-Word Title Matches
 
+ENC_SEARCH_ARTICLE_MATCHES
+Article Description Matches
```

Comment:
"Article Description" is misleading... I think you mean article contents or text?